ui-v2/package.json

```
@@ -13,7 +13,9 @@
 		"format:check": "biome check",
 		"format": "biome check --write",
 		"preview": "vite preview",
-		"service-sync": "uv run ../scripts/generate_oss_openapi_schema.py && npx openapi-typescript oss_schema.json -o src/api/prefect.ts && rm oss_schema.json",
+		"generate:types": "uv run --with-editable ../. ../scripts/generate_oss_openapi_schema.py && npx openapi-typescript oss_schema.json -o src/api/prefect.ts && rm oss_schema.json",
```

We remove this in https://github.com/PrefectHQ/prefect/pull/17400 and want to make sure we don't reintroduce it
```suggestion
		"generate:types": "uv run ../scripts/generate_oss_openapi_schema.py && npx openapi-typescript oss_schema.json -o src/api/prefect.ts && rm oss_schema.json",
```